src/App.js

```diff
@@ -13,16 +13,15 @@ const manifolds = {};
 let dirty = false;
 
 function bytesToNumber(buffer) {
+//  const bytes = new Uint8Array(buffer);
```

Comment:
Looks like you don't need this commented out line since there is an uncommented copy right below it.